### PR TITLE
Document the format property of the date and date_range controls.

### LINKED ISSUE #772: Add examples for date format in docs
Both date and date_range visual elements' docs do not provide examples for the format property which is supposed to define the date format. This leads to users not being able to choose the date format.
https://docs.taipy.io/en/develop/manuals/gui/viselements/date/
https://docs.taipy.io/en/develop/manuals/gui/viselements/date_range/

### PR DESCRIPTION
**Pull request by FabienLelaquais**
Resolves [taipy#772](https://github.com/Avaiga/taipy/issues/772).

### CODE REVIEW

File: docs/manuals/gui/viselements/date.md_template [hunk 1]
```diff
@@ -15,18 +25,86 @@ The [Stylekit](../styling/stylekit.md) provides a specific class that you can us
...
+```python
+import datetime
+
+date = datetime.datetime(1789, 7, 14, 17, 5, 12)
+```
+
+You can create a date selector that represents it with the following definition:
+
+!!! taipy-element
+    default={date}
```
Comment (FredLL-Avaiga) [inline docs/manuals/gui/viselements/date.md_template:48]: default ? or date ?

Comment (FredLL-Avaiga) [reply docs/manuals/gui/viselements/date.md_template:48]: or is it some magic to generate the right example in different formats ?

**Review by FredLL-Avaiga — APPROVED** [review]
never mind my comment

File: docs/manuals/gui/viselements/date.md_template (hunk 1 above)
Comment (FabienLelaquais) [reply docs/manuals/gui/viselements/date.md_template:48]: Indeed there is.

**Pull request merged**